src/server/ua_services_nodemanagement.c

```diff
@@ -823,7 +823,6 @@ void Service_AddReferences(UA_Server *server, UA_Session *session, const UA_AddR
 	}
 #endif
 
-	response->resultsSize = size;
```

Comment:
This is already set a few lines before and `size` or `resultsSize` should not change in between?!